src/components/structural/header/SceneConfigMenu.js

```diff
@@ -423,6 +424,29 @@ class ConfigModal extends Component {
         );
     };
 
+    /**
+     * Returns button for toggle cursor
+     */
+    cursorToggle = () => {
+        let style = this.props.scene.settings.showCursor ? btnStyle.on : btnStyle.off;
+        style = { ...btnStyle.base, ...style };
+        return (
+            <ButtonBase
+                style={style}
+                onClick={() => {
+                    this.props.handleRender();
+                    this.props.sceneActions.toggleCursor();
+                }} >
+                {
+                    this.props.scene.settings.showCursor
+                        ? <Icon className="material-icons">toggle_on</Icon>
```

Comment:
Could you change the icon to something that represents the action of "resetting" or "clearing"?

Example: https://mui.com/material-ui/material-icons/?query=restart&selected=RestartAlt